morphos/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: morphos
+category: files
+name: Morphos Server
+version: "0.1.1"
+tagline: Self-hosted file converter server
+description: >-
+  Today we are forced to rely on third party services to convert files to other formats. This is a serious threat to our privacy, if we use such services to convert files with  highly sensitive personal data. It can be used against us, sooner or later.
+
+  Morphos server aims to solve the problem mentioned above, by providing a self-hosted server to convert files privately. The project provides an user-friendly web UI.
+
+  For now, Morphos only supports images. Documents will be added soon.
```

Comment:
Rendering the yaml nicely is a bit tricky. To get a blank line for whitespace between paragraphs you need to add **2** blank lines in the yaml.

Otherwise it looks like this:

<img width="713" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/cb723805-4177-43cf-be3d-71185abb026e">

Reply:
I just rephrased the half of the description added a new list of feature, and of course, extra lines for better rendering.